app/src/main/res/layout/activity_post_requirement.xml

```diff
@@ -28,9 +28,10 @@
             android:orientation="horizontal">
 
             <TextView
+                android:id="@+id/textViewTitle"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:text="New Post"
+                android:text="New Request"
```

Comment:
Shouldn't this be a generalized "Post" rather than "Request" since we also have "Offerings"?

Reply:
Can you add a bit more padding on top of the header? @akhillllldev It seems to hit the edges.